.github/dependabot.yml

```diff
@@ -1,68 +0,0 @@
----
-version: 2
-updates:
-  - package-ecosystem: "github-actions"
-    directory: "/"
-    schedule:
-      interval: "daily"
-  - package-ecosystem: "maven"
-    directory: "/"
-    schedule:
-      interval: "daily"
-    ignore:
-      # Exclusions in this section have been triaged and determined to be
-      # permanent. We do not anticipate removing exclusions from this section.
-
-      # Provided by Jetty and should be aligned with the version provided by the
-      # version of Jetty we deliver. See:
-      # https://github.com/jenkinsci/jenkins/pull/5211
-      - dependency-name: "jakarta.servlet:jakarta.servlet-api"
-
-      # Jetty Maven Plugin and Winstone should be upgraded in lockstep in order
-      # to keep their corresponding Jetty versions aligned.
-      - dependency-name: "org.eclipse.jetty:jetty-maven-plugin"
-      - dependency-name: "org.jenkins-ci:winstone"
-
-      # Here lies technical debt. Exclusions in this section have been triaged
-      # and determined to be temporary. Exclusions should be removed from this
-      # section once the remaining action items have been completed.
-
-      # Contains incompatible API changes and needs compatibility work.
-      - dependency-name: "jakarta.servlet.jsp.jstl:jakarta.servlet.jsp.jstl-api"
-
-      # Needs significant testing. See:
-      # https://github.com/jenkinsci/jenkins/pull/5112#issuecomment-744429487
-      # https://github.com/jenkinsci/jenkins/pull/5116#issuecomment-744526638
-      - dependency-name: "org.codehaus.groovy:groovy-all"
-        versions: [">=2.5.0"]
-
-      # Consumed by Groovy and should be updated in lockstep with Groovy. See:
-      # https://github.com/jenkinsci/jenkins/pull/5184
-      - dependency-name: "org.fusesource.jansi:jansi"
-
-      # Contains incompatible API changes and needs compatibility work. See:
-      # https://github.com/jenkinsci/jenkins/pull/4224
-      - dependency-name: "org.jfree:jfreechart"
-
-      # Starting with 6.x, Spring requires Java 17 at a minimum.
-      - dependency-name: "org.springframework:spring-framework-bom"
-        versions: [">=6.0.0"]
-
-      # Starting with 6.x, Spring Security requires Java 17 at a minimum.
-      - dependency-name: "org.springframework.security:spring-security-bom"
-        versions: [">=6.0.0"]
-
-      # Starting with 7.x, Guice switches from javax.* to jakarta.* bindings.
-      # See https://github.com/google/guice/wiki/Guice700
-      - dependency-name: "com.google.inject:guice-bom"
-        versions: [">=7.0.0"]
-  - package-ecosystem: "maven"
-    directory: "/"
-    target-branch: "stable-2.452"
-    labels:
-      - "into-lts"
-      - "needs-justification"
-    schedule:
-      interval: "daily"
-    # Include only security updates and exclude version updates.
-    open-pull-requests-limit: 0
```

Comment:
I don't think this ever worked, see https://github.com/jenkinsci/jenkins/pulls?q=sort%3Aupdated-desc+is%3Apr+is%3Aclosed+author%3Aapp%2Fdependabot+label%3Ainto-lts

so I haven't kept it for going forward

Reply:
> I don't think this ever worked

Sad, given that it cost me so much to get that change accepted. I still think this is something that we want to automated, and we essentially do it manually for every LTS release, including recent ones.

Reply:
Restored in https://github.com/jenkinsci/jenkins/pull/9459/commits/74eee38e3652690d828ff081db16f78abd5b1bfd

had to debug a fair but and used:
```
RENOVATE_CONFIG_FILE=.github/renovate.json renovate --token=$GITHUB_TOKEN --schedule=""  --require-config=ignored --dry-run=full timja-org/jenkins
```

Reply:
Here are the LTS PRs that would be created: https://github.com/timja-org/jenkins/pulls?q=sort%3Aupdated-desc+is%3Apr+is%3Aopen+label%3Ainto-lts

(some remoting ones would disappear by backporting this PR to LTS)